sdk/attestation/azure-security-attestation/README.md

```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
 
 ## Examples
 
-- [Instantiate a synchronous attestation client](#create-a-synchronous-attestation-client)
-- [Retrieve token validation certificates](#retrieve-token-certificates)
+- [Create a synchronous attestation client](#create-a-synchronous-attestation-client)
+- [Retrieve Token Certificates](#retrieve-token-certificates)
```

Comment:
Change the anchor, not the link for `Retrieve Token Certificates`. The old name is better.